frontend/src/scenes/experiments/ExperimentView/ProgressBar.tsx

```diff
@@ -0,0 +1,77 @@
+import '../Experiment.scss'
+
+import { useValues } from 'kea'
+import { dayjs } from 'lib/dayjs'
+import { LemonProgress } from 'lib/lemon-ui/LemonProgress'
+import { humanFriendlyNumber } from 'lib/utils'
+
+import { FunnelStep, InsightType } from '~/types'
+
+import { experimentLogic } from '../experimentLogic'
+
+export function ProgressBar(): JSX.Element {
+    const { experiment, experimentResults, experimentInsightType } = useValues(experimentLogic)
+
+    // Parameters for experiment results
+    // don't use creation variables in results
+    const funnelResultsPersonsTotal =
+        experimentInsightType === InsightType.FUNNELS && experimentResults?.insight
+            ? (experimentResults.insight as FunnelStep[][]).reduce(
+                  (sum: number, variantResult: FunnelStep[]) => variantResult[0]?.count + sum,
+                  0
+              )
+            : 0
+
+    const experimentProgressPercent =
+        experimentInsightType === InsightType.FUNNELS
+            ? ((funnelResultsPersonsTotal || 0) / (experiment?.parameters?.recommended_sample_size || 1)) * 100
+            : (dayjs().diff(experiment?.start_date, 'day') / (experiment?.parameters?.recommended_running_time || 1)) *
+              100
+
+    return (
+        <div>
+            <div className="mb-1 font-semibold">{`${
+                experimentProgressPercent > 100 ? 100 : experimentProgressPercent
+            }% complete`}</div>
+            <LemonProgress
+                className="w-full border"
+                bgColor="var(--bg-table)"
+                size="large"
+                percent={experimentProgressPercent}
```

Comment:
<img width="295" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/d71f2139-8011-4577-970d-fb29945715b8">

Not clamping appropriately